### DISCUSSION TITLE
Add ILS (₪) as a currency format

### DISCUSSION THREAD

**Discussion by itaispiegel** [grafana/grafana#37897] [Dashboards, Panels & Visualizations]
The Israeli new shekel isn't a possible currency format.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
